core/spec.md

```diff
@@ -1097,14 +1097,14 @@ Where:
 - The `COLLECTIONScount` attribute MUST contain the number of entities in the
   `COLLECTIONS` map (after any necessary [filtering](#filter)) and MUST
   be a read-only attribute that MUST be silently ignored by a server during
-  an write operation.
+  a write operation.
 - The `COLLECTIONS` attribute is a map and MUST contain the entities of the
   collection (after any necessary [filtering](#filter)), and MUST use
   the `SINGULARid` of each entity as its map key.
 - The key of each entity in the collection MUST be unique within the scope of
   the collection.
 - The specifics of whether each attribute is REQUIRED or OPTIONAL will be
-  based whether document or API view is being used - see the next section.
+   based on whether a document or API view is being used - see the next section.
```

Comment:
not sure about the "a" here because "document" isn't referring to an actual "document" but rather to "document "mode". so it's "doc or API view" (or mode). Make sense?